development/prs_and_reviewing.ipynb

```diff
@@ -226,7 +226,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.1"
+   "version": "3.8.5"
```

Comment:
versions increase linearly, no? this was an arbitrary number, not sure you needed to change this